From e70a7f4c7d16680d159f08decc1c04f5c13067aa Mon Sep 17 00:00:00 2001
From: Jeremie Graulle <jeremiex.graulle@intel.com>
Date: Tue, 2 Apr 2013 12:57:58 +0200
Subject: libmedia_helper: System include for host

BZ: 93434
[CLEAN]

There is a compilation error when the host is NULL.

This fix adds the NULL macro through stdlib library.

Category: aosp improvement
Domain: AUDIO_MEDIA-Audio Framework
Origin: internal
Upstream-Candidate: yes
Change-Id: Id7c8f9961cfd439e6916dcf169b7fcd12cd34f2b
Orig-Change-Id: Ic330be220aa36525c75b2b2f268bfd8c2c1874b8
Signed-off-by: Jeremie Graulle <jeremiex.graulle@intel.com>
---
 include/media/AudioBufferProvider.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/media/AudioBufferProvider.h b/include/media/AudioBufferProvider.h
index ef392f0..c26b83f 100644
--- a/include/media/AudioBufferProvider.h
+++ b/include/media/AudioBufferProvider.h
@@ -18,6 +18,7 @@
 #define ANDROID_AUDIO_BUFFER_PROVIDER_H
 
 #include <utils/Errors.h>
+#include <stdlib.h>
 
 namespace android {
 // ----------------------------------------------------------------------------
-- 
1.7.4.1

